plugins/apps/clementine.mk

```diff
@@ -0,0 +1,45 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := clementine
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 1.3
+$(PKG)_CHECKSUM := 55e8d283ed02e3f2376a88b012cd2003e00fef95a98b6a919d67cb57e96b8617
+$(PKG)_SUBDIR   := Clementine-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://github.com/$(PKG)-player/$(PKG)/archive/$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc boost chromaprint cryptopp dlfcn-win32 fftw glew gst-plugins-bad \
+                   gst-plugins-good gst-plugins-ugly libarchive libechonest libid3tag \
+                   libmpcdec libplist libusb1 protobuf qtsparkle_qt4 sparsehash
+
+define $(PKG)_UPDATE
+    $(call MXE_GET_GITHUB_TAGS, clementine-player/clementine)
+endef
+
+define $(PKG)_BUILD
+    cd '$(BUILD_DIR)' && '$(TARGET)-cmake' '$(SOURCE_DIR)' \
+        -DCMAKE_INSTALL_PREFIX=$(PREFIX)/$(TARGET)/apps/$(PKG)
+    $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
+    $(MAKE) -C '$(BUILD_DIR)' -j 1 install
+
+    $(if $(BUILD_SHARED),
+        $(INSTALL) -d '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)'/bin/{libgstapetag,libgstapp,libgstasf,libgstaudioconvert,libgstaudiofx,libgstaudioparsers,libgstaudioresample,libgstaudiotestsrc,libgstautodetect,libgstcdio,libgstcoreelements,libgstdirectsoundsink,libgstequalizer,libgstfaad,libgstflac,libgstgdp,libgstgio,libgsticydemux,libgstid3demux,libgstisomp4,libgstlame,libgstmad,libgstogg,libgstopus,libgstplayback,libgstreplaygain,libgstsouphttpsrc,libgstspectrum,libgstspeex,libgsttaglib,libgsttcp,libgsttypefindfunctions,libgstudp,libgstvolume,libgstvorbis,libgstwavpack,libgstwavparse}.dll \
+        '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
```

Comment:
Please, indent this line, because it is a continuation of the command started on the previous line.
Also remove `;` here and on other lines, where it is not needed.
Also do the same with commands below (installing files to imageformats directory).